apps/crn-frontend/src/AuthenticatedApp.tsx

```diff
@@ -95,6 +100,11 @@ const AuthenticatedApp: FC<Record<string, never>> = () => {
                   <Discover />
                 </Frame>
               </Route>
+              <Route path={tutorials.template}>
+                <Frame title="Tutorials">
+                  <DiscoverTutorials />
```

Comment:
I think this could just be called Tutorials. I don't think there will be any other type of tutorials